cmd/thermal-recorder/cptvfilerecorder.go

```diff
@@ -138,6 +140,7 @@ func deleteTempFiles(directory string) error {
 }
 
 func checkDiskSpace(mb uint64, dir string) (bool, error) {
+	// These are 'Nix specific calls.  For testing on Windows, just return true, nil.
```

Comment:
Yes but I still can't compile and run on Windows say, because those Unix system calls are not defined.

And it will be the same on Unix I imagine, the Windows ones won't be defined.

Reply:
I will read up on conditional compilation.

Reply:
Menno, I pushed the changes I have made today.

I have just taken my comment out of that checkDiskSpace function.

Because making it work for both Windows and Linux seems like quite a lot of work, and the executable will probably never run on Windows I guess.

I read a bit about conditional compilation in Go and it seems you have to have different source files.  And either use tags or file extensions to tell the compiler which ones to build.  But then we'd have to have 2 files containing that function and maintain 2 etc.

Reply:
Yep, you have to use separate files with appropriate name suffixes or build tags. 

It's actually not that hard, but of limited value in the case.